### DISCUSSION TITLE
build some project use goreleaser in my meachine,it's size so big, reach 20MB +, but download binary from from github release,it's so tiny, just 7MB

### DISCUSSION THREAD

**Discussion by Fly-Playgroud** [goreleaser/goreleaser#3784] [Q&A] [answered]
build this project use goreleaser in my meachine,it's size so big, reach 20MB +, but download binary from from github release,it's so tiny, just 7MB.why cause this deference

**Comment by caarlos0** [comment, marked as answer]
probably different build flags, run `goreleaser build --clean --debug --single-target --snapshot` and you'll see..

but probably your config has the strip debug symbols et al (`go build -ldflags="-s -w"`)
